cfgov/v1/models/organisms.py

```diff
@@ -22,6 +22,22 @@ class Meta:
         template = '_includes/organisms/full-width-text.html'
 
 
+class ImageText5050Group(blocks.StructBlock):
+    image_texts = blocks.ListBlock(molecules.ImageText5050())
```

Comment:
If they are always groups then this should also be the case for Image Text 25 75, right? @KimberlyMunoz